.github/workflows/composite/bazel-gh-cache/action.yml

```diff
@@ -0,0 +1,66 @@
+# Copyright 2023 The Magma Authors.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Update Bazel Caches on GitHub
+description: Pulls caches for .bazel-cache and .bazel-cache-repo for a given key prefix and purges them based on a threshold
+
+inputs:
+  cache-key-prefix:
+    description: A prefix used for cache keys
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Set variables
```

Comment:
Note, I don't like this, but composite actions do not seem to support the scoped env directive outside of steps. I assume these variables are now also accessible by the calling workflows - this is not an issue per se, but not a good scoping pattern.

Reply:
A possible workaround for this would be to use optional inputs like
```
inputs:
  BAZEL_CACHE_NAME:
    description: Name of the Bazel cache folder
    required: false
    default: "bazel-cache"
```
this would also make the cut-offs variable.

Reply:
Else I would prefer not to use the step env and directly enter the values in the run command, but both are not very elegant unfortunately.

Reply:
Compared with you suggestions, I prefer the current "Set variables" step. This still has the benefit to have constants defined sufficiently near to the header.